velox/exec/TableScan.cpp

```diff
@@ -256,6 +256,7 @@ void TableScan::preload(std::shared_ptr<connector::ConnectorSplit> split) {
        ctx = operatorCtx_->createConnectorQueryCtx(
            split->connectorId, planNodeId(), connectorPool_),
        task = operatorCtx_->task(),
+       pendingDynamicFilters = pendingDynamicFilters_,
```

Comment:
Just do `std::move(pendingDynamicFilters = pendingDynamicFilters_)`

Reply:
I think you mean we should move not copy the `pendingDynamicFilters_`. maybe we should use by-reference capture here.